cognite/client/_api/transformation_notifications.py

```diff
@@ -0,0 +1,117 @@
+        """`Subscribe for notifications on the transformation errors. <https://docs.cognite.com/api/playground/#operation/subscribeTransformationNotifications>`_
+
+        Args:
+            notification (Union[TransformationNotification, List[TransformationNotification]]): Notification or list of notifications to create.
+
+        Returns:
+            Created notification(s)
+
+        Examples:
+
+            Create new notifications:
+
+                >>> from cognite.experimental import CogniteClient
+                >>> from cognite.experimental.data_classes import TransformationNotification
+                >>> c = CogniteClient()
+                >>> notifications = [TransformationNotification(transformation_id = 1, destination="my@email.com"), TransformationNotification(transformation_external_id="transformation2", destination="other@email.com"))]
+                >>> res = c.transformations.notifications.create(notifications)
+        """
+        utils._auxiliary.assert_type(notification, "notification", [TransformationNotification, list])
+        return self._create_multiple(notification)
+
+    def list(
+        self,
+        transformation_id: Optional[int] = None,
+        transformation_external_id: str = None,
+        destination: str = None,
+        limit: Optional[int] = 25,
+    ) -> TransformationNotificationList:
+        """`List notification subscriptions. <https://docs.cognite.com/api/playground/#operation/listTransformationNotifications>`_
+
+        Args:
+            transformation_id (Optional[int]): List only notifications for the specified transformation. The transformation is identified by internal numeric ID.
+            transformation_external_id (str): List only notifications for the specified transformation. The transformation is identified by externalId.
+            destination (str): Filter by notification destination.
+            limit (int): Limits the number of results to be returned. To retrieve all results use limit=-1, default limit is 25.
+
+        Returns:
+            TransformationNotificationList: List of transformation notifications
+
+        Example:
+
+            List all notifications::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> notifications_list = c.transformations.notifications.list()
+
+            List all notifications by transformation id::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> notifications_list = c.transformations.notifications.list(transformation_id = 1)
+
+            List all notifications by transformation external id::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> notifications_list = c.transformations.notifications.list(transformation_external_id = "myExternalId")
```

Comment:
can remove the first example, looks redundant.

Reply:
transformation notification does not have external id